app/guid-file/template.hbs

```diff
@@ -66,11 +66,11 @@
             </div>
         {{/if}}
         {{#if this.metadataOpened}}
-            <div local-class='right-container {{if this.isMobile 'Mobile'}}'>
+            <div local-class='right-container {{if this.isMobile 'Mobile'}}' data-test-metadata-tab>
                 <FileMetadataManager @file={{this.model.fileModel}} as |manager|>
                     <div local-class='metadata-title-edit-download'>
-                        <h2 local-class='metadata-heading'>
-                            {{t 'file-detail.file-metadata'}}
+                        <h2 local-class='metadata-heading' data-test-metadata-header>
+                            {{t 'file_detail.file_metadata'}}
```

Comment:
All of the underscores in this file's translation keys need to be turned into hyphens. For example, 
```suggestion
                            {{t 'file-detail.file-metadata'}}
```